src/Http/Endpoint.php

```diff
@@ -46,6 +48,13 @@ public function process(
             return (new \LORIS\Middleware\ETag())->process($request, $handler);
         }
 
+        $loris    = $request->getAttribute('loris');
+        $loglevel = $loris->getConfiguration()
+            ->getLogSettings()
+            ->getRequestLogLevel();
+
+        $this->logger = new \LORIS\Log\ErrorLogLogger($loglevel);
+
```

Comment:
This is going to get bypassed if the if statement above it gets entered.

Reply:
moved it above the if statement. Thanks!